server/routes/main/refresh.js

```diff
@@ -0,0 +1,16 @@
+var querystring = require("querystring");
+
+module.exports = function(config, req, res) {
+  var qs = querystring.stringify(req.query);
+  if(qs !== "") {
+    qs = "?" + qs;
+  }
+
+  var options = {
```

Comment:
nit: pretty sure you don't need any of these since you don't use them in the rendered template

Reply:
which also means you don't need the `qs` stuff above either

Reply:
OK will update.